src/DssSpell.sol

```diff
@@ -16,234 +16,99 @@
-        // https://vote.makerdao.com/polling/QmUx9LVs?network=mainnet#vote-breakdown
-
-        (,,,line,) = vat.ilks("LRC-A");
-        totalLineReduction = _add(totalLineReduction, line);
-        DssExecLib.setIlkLiquidationPenalty("LRC-A", 0);
-        DssExecLib.removeIlkFromAutoLine("LRC-A");
-        DssExecLib.setIlkDebtCeiling("LRC-A", 0);
-        DssExecLib.linearInterpolation({
-            _name:      "LRC Offboarding",
-            _target:    DssExecLib.spotter(),
-            _ilk:       "LRC-A",
-            _what:      "mat",
-            _startTime: block.timestamp,
-            _start:     CURRENT_LRC_MAT,
-            _end:       TARGET_LRC_MAT,
-            _duration:  60 days
-        });
 
-        // Offboard ZRX-A 
-        // https://vote.makerdao.com/polling/QmPfuF2W?network=mainnet#vote-breakdown
-
-        (,,,line,) = vat.ilks("ZRX-A");
-        totalLineReduction = _add(totalLineReduction, line);
-        DssExecLib.setIlkLiquidationPenalty("ZRX-A", 0);
-        DssExecLib.removeIlkFromAutoLine("ZRX-A");
-        DssExecLib.setIlkDebtCeiling("ZRX-A", 0);
-        DssExecLib.linearInterpolation({
-            _name:      "ZRX Offboarding",
-            _target:    DssExecLib.spotter(),
-            _ilk:       "ZRX-A",
-            _what:      "mat",
-            _startTime: block.timestamp,
-            _start:     CURRENT_ZRX_MAT,
-            _end:       TARGET_ZRX_MAT,
-            _duration:  60 days
+        // Increase the GUNIV3DAIUSDC1-A Debt Ceiling - October 11, 2021
+        //  https://vote.makerdao.com/polling/QmU6fTQx?network=mainnet#poll-detail
+        DssExecLib.setIlkAutoLineParameters("GUNIV3DAIUSDC1-A", 50 * MILLION, 10 * MILLION, 8 hours);
+
+
+        // Parameter Changes Proposal - MakerDAO Open Market Committee - October 18, 2021
+        //  https://vote.makerdao.com/polling/QmP6GPeK?network=mainnet#poll-detail
+        DssExecLib.setIlkAutoLineParameters("ETH-A", 15_000 * MILLION, 150 * MILLION, 6 hours);
+        DssExecLib.setIlkAutoLineParameters("ETH-B", 500 * MILLION, 20 * MILLION, 6 hours);
+        DssExecLib.setIlkAutoLineParameters("WBTC-A", 1_500 * MILLION, 60 * MILLION, 6 hours);
+
+
+        // Add stETH (Lido Staked ETH) as a new Vault Type - October 11, 2021
+        //  https://vote.makerdao.com/polling/QmXXHpYi?network=mainnet#poll-detail
+        //  https://forum.makerdao.com/t/steth-collateral-onboarding-risk-evaluation/9061
+        CollateralOpts memory WSTETH_A = CollateralOpts({
+            ilk:                   "WSTETH-A",
+            gem:                   WSTETH_GEM,
+            join:                  MCD_JOIN_WSTETH_A,
+            clip:                  MCD_CLIP_WSTETH_A,
+            calc:                  MCD_CLIP_CALC_WSTETH_A,
+            pip:                   PIP_WSTETH,
+            isLiquidatable:        true,
+            isOSM:                 true,
+            whitelistOSM:          true,
+            ilkDebtCeiling:        5 * MILLION,
```

Comment:
Yeah, I think I went with our historical precedent here but the execLib doesn't call the autoline module so this will be 5mm until it's called. I'll update this and add a comment that the real debt ceiling is getting added via the autoline.